controller/controller.go

```diff
@@ -1449,20 +1454,7 @@ func (ctrl *ProvisionController) deleteVolumeOperation(ctx context.Context, volu
 	operation := fmt.Sprintf("delete %q", volume.Name)
 	glog.Info(logOperation(operation, "started"))
 
-	// This method may have been waiting for a volume lock for some time.
```

Comment:
Removed this due to the earlier discussion on locks. `deleteVolumeOperation` takes the `volume` object as an input parameter